src/python/pants/backend/go/util_rules/first_party_pkg.py

```diff
@@ -234,12 +234,15 @@ async def compute_first_party_package_import_path(
 
 @rule
 async def analyze_first_party_package(
-    request: FirstPartyPkgAnalysisRequest,
-    analyzer: PackageAnalyzerSetup,
-    golang_subsystem: GolangSubsystem,
+    request: FirstPartyPkgAnalysisRequest, analyzer: PackageAnalyzerSetup
 ) -> FallibleFirstPartyPkgAnalysis:
     wrapped_target, import_path_info, owning_go_mod = await MultiGet(
-        Get(WrappedTarget, Address, request.address),
+        Get(
+            WrappedTarget,
+            WrappedTargetRequest(
+                request.address, description_of_origin="<first party pkg analysis>"
```

Comment:
I want to try redesigning the API so that there's no need to request the `WrappedTarget` - we are given everything we need.